README.md

```diff
@@ -124,6 +124,9 @@ app.get("/api/protected",
 );
 ```
 The scope parameter defines the required scopes, and the audience parameter is the resource URI.
```

Comment:
update to :
The scope parameter defines the required scopes, and the audience parameter is the application clientId or the resource URI.

Reply:
comma before and (Oxford annotation ?)